src/components/layout/CyVerseAppBar.js

```diff
@@ -122,18 +127,156 @@ function CyverseAppBar(props) {
             throwOnError: true,
         },
     });
-    React.useEffect(() => {
-        if (enabled && userProfile && userProfile.id) {
-            intercomLogin(
-                userProfile.id,
-                userProfile.attributes.email,
-                appId,
-                companyId,
-                companyName
+
+    const gotToOutputFolder = useCallback(
+        (outputFolder) => {
+            router.push(
+                `${constants.PATH_SEPARATOR}${NavigationConstants.DATA}${constants.PATH_SEPARATOR}ds${outputFolder}`
```

Comment:
Not really for this PR, but it'd be nice to get something set up so these router pushs don't need to happen at the component level.  🤔